obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  mp_starcat_time: 2006:212:07:39:07.107
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:05:35:23.362
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:06:30:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:07:13:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:07:56:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:08:39:11.687
+  status: 1
+- agasc_id:
+  - 1117787424
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2006:220:09:22:11.687
+  status: 1
+- agasc_id:
+  - 879768704
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2007:085:18:25:58.626
+  status: 1
+- agasc_id:
+  - 1020796600
+  comments: Never acquired; set mag 11.5 +/- 0.1
+  mp_starcat_time: 2007:130:19:34:45.821
+  status: 1
+- agasc_id:
+  - 676336280
+  comments: Available data look OK, lack of track not understood
+  mp_starcat_time: 2007:231:04:53:28.518
+  status: 0
+- agasc_id:
+  - 38535536
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2008:057:22:07:34.020
+  status: 1
+- agasc_id:
+  - 185871616
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2008:073:16:53:06.691
+  status: 1
+- agasc_id:
+  - 974791856
+  comments: Faint star. Poorly tracked. Set mag to MAXMAG
+  mp_starcat_time: 2008:183:05:33:47.836
+  status: 1
+- agasc_id:
+  - 978846152
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978848616
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978849272
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978851616
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978853160
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 978986608
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 1043726440
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 1043726952
+  comments: Normal Sun Mode
+  mp_starcat_time: 2008:225:09:21:55.521
+  status: 1
+- agasc_id:
+  - 639635448
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:023:22:13:56.794
+  status: 1
+- agasc_id:
+  - 639635448
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2009:024:04:12:45.955
+  status: 1
+- agasc_id:
+  - 395055600
+  comments: UNKNOWN Never acquired. Unknown issue with observation. Presume star is OK.
```

Comment:
There's a web page for obsid 9931 at https://cxc.harvard.edu/mta/ASPECT/obs9931_drops/ so yes, star is probably fine.